src/katsdpcontroller/product_controller.py

```diff
@@ -2134,6 +2156,26 @@ async def request_beam_delays(self, ctx, stream: str, *coefficient_sets: str) ->
         """
         await self._get_product().beam_delays(stream, coefficient_sets)
 
+    async def request_dsim_signals(
+        self, ctx, dsim: str, signals_str: str, period: Optional[int] = None
+    ) -> int:
+        """Update the signals that are generated by a dsim.
+
+        Parameters
+        ----------
+        dsim
+            The task name for the dsim (implementation-defined).
+        signals_str
+            Textual description of the signals. Refer to the katgpucbf manual
+            for details. The description must produce one signal per polarisation.
+
+        Returns
+        -------
+        timestamp
+            First timestamp which will use the new signals
```

Comment:
Full stop at the end of this sentence, please. Just to be consistent with your additions above.